hack/update-payload-crds.sh

```diff
@@ -26,6 +26,8 @@ crd_globs="\
     config/v1alpha1/zz_generated.crd-manifests/0000_10_config-operator_01_clustermonitoring*.crd.yaml
     operator/v1/zz_generated.crd-manifests/*_storage_01_storages*.crd.yaml
     operator/v1/zz_generated.crd-manifests/*_csi-driver_01_clustercsidrivers*.crd.yaml
+    insights/v1/zz_generated.crd-manifests/0000_10_insights_01_datagathers*.crd.yaml
+    config/v1/zz_generated.crd-manifests/0000_10_config-operator_01_insightsdatagathers*.crd.yaml
```

Comment:
I think you want the opposite of this. If your controllers are using the `v1alpha1` version of the API, you'll want to make sure that the `v1alpha1` version of the API is the one present in the payload. As-is, this looks like it will put the `v1` version in the payload which will continue to cause the CI failures you've been seeing related to the insights-operator.